extensions/tynyfy/src/tynyfy.tsx

```diff
@@ -0,0 +1,129 @@
+import { Action, ActionPanel, Form, Clipboard, showHUD, showToast, Toast, popToRoot } from "@raycast/api";
+import { useEffect, useRef, useState } from "react";
+import fetch from "node-fetch";
+import { APIs } from "./data/apiData";
+import { isValidUrl } from "./helpers/url";
+
```

Comment:
You can probably use our [useForm hook](https://developers.raycast.com/utilities/react-hooks/useform) from `@raycast/utils`. This will make it easier to provide validation and errors, but also handling state for your form.

Reply:
thanks for this. I didn't know about this. but I think the form is simple enough with minimal state and validation so I don't see any value for a custom hook here.

Reply:
The hook is quite easy to use and has minimal code. Providing validation and error on fields without this hook usually is more of a hassle, but it's up to you. 😊